hermes/apps/athena/modules/Earthfile

```diff
@@ -78,15 +79,35 @@ get-local-hermes:
 # Input: WIT interface definition files
 # Output: hermes.rs - Generated Rust binding code (shared by all modules)
 # Duration: ~1-2 minutes (WIT processing and code generation)
-gen-bindings:
+gen-bindings-http-proxy:
     # Use the WASI build environment which includes WIT binding generation tools
     FROM wasi+build-rust-bindings
 
     # Export generated bindings to the HTTP proxy module source directory
     # This allows the Rust code in http-proxy/src/ to import and use Hermes APIs
-    # NOTE: In multi-module apps, this same hermes.rs is copied to each module's src/
     SAVE ARTIFACT hermes.rs AS LOCAL athena/modules/http-proxy/src/hermes.rs
 
+builder-db:
```

Comment:
I think it is better to move this module specific things into its Earthfile so we wont end up having large Earthfile